lib/create.js

```diff
@@ -68,7 +68,6 @@ var create = module.exports = function(templateName, config, templateModule) {
     });
 
 };
-exports.create = create;
```

Comment:
Either this needs to be reinstated or line 82 needs to be updated to use `create` instead of `exports.create` https://github.com/montagejs/minit/blob/36c4359794b9c1d6825d0b9605b1b620195ef99f/lib/create.js#L82